src/smart-components/Channel/components/MessageInput/index.tsx

```diff
@@ -33,14 +40,49 @@ const MessageInputWrapper = (): JSX.Element => {
   const isOperator = utils.isOperator(channel);
   const { isBroadcast } = channel;
 
+  useEffect(() => {
+    if (mentionedUsers?.length >= 10) {
```

Comment:
what's the reason for `setDisableMention ` when mentionedUsers changes?

Reply:
It's for mention configuration, we will allow to set up a maximum mentionable count as a configuration